Automattic/wp-calypso
# Untangling: apply new design to Settings -> Administration -> Manage connection

Fixes https://github.com/Automattic/dotcom-forge/issues/9864

## Proposed Changes

This PR applies the new global view design to Settings -> Administration -> Manage connection flow.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Prepare a fresh Jurassic Ninja site.
2. Connect it with Jetpack.
3. Go to DIRECT LINK (for now) `/sites/settings/administration/:siteSlug`
4. Click the `Manage connection` card, verify you see:

<img width="997" alt="image" src="https://github.com/user-attachments/assets/68a9ad0d-1c26-4894-bbe8-e07371bb86f9">

5. Try clicking around.
6. REGRESSION: Test also `/settings/general/:siteSlug` and click Manage connection at the bottom.

### NOTE

I am not touching this Disconnect Jetpack flow because it's already "full-screen" and not that important in terms of Untangling Hosting Menu.

<img width="1006" alt="image" src="https://github.com/user-attachments/assets/fe94ecf5-a476-4f4f-962a-751d7c29731f">

## Pre-merge Checklist
